torch/csrc/jit/script/parser.h

```diff
@@ -266,6 +283,9 @@ struct Parser {
         auto values = parseList(TK_NOTHING, ',', TK_NEWLINE, &Parser::parseExp);
         return Return::create(range, values);
       }
+      case TK_TYPE_COMMENT: {
```

Comment:
A type comment can only appear at the beginning of a function so it should not be parsed here.